composer.json

```diff
@@ -22,6 +22,7 @@
         "drupal/entityqueue": "1.x-dev",
         "drupal/fontyourface": "3.x-dev",
         "drupal/google_analytics": "^3.0",
```

Comment:
we forgot to composer remove drupal/google_analytics.  do that and push again please.